### PR TITLE
[mtouch] Remove warning when building mtouch.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

File: tools/common/MachO.cs [hunk 1]
```diff
@@ -812,7 +812,6 @@ public override void Dump ()
 	}
 	
 	public class UuidCommand : LoadCommand {
-		public uint cmd;
 		public uint cmdsize;
```
Comment (spouliot) [inline tools/common/MachO.cs]: `cmdsize` is also defined in `LoadCommand` so you'll get the same warning

**Comment by spouliot** [comment]
Also please using tags for the component, e.g. `[mtouch]`, and not something like `[Fix]` as the rest of the message should describe what's being done.

**Comment by spouliot** [comment]
LGTM, just confirm there's no other warning in Jenkins build logs before merging.

**Comment by xamarin-release-manager** [comment]
Build success

**Comment by xamarin-release-manager** [comment]
Build success

**Comment by xamarin-release-manager** [comment]
Build failure

**Comment by mandel-macaque** [comment]
@spouliot looking at the logs, we are ok. Although the last one failed and I believe is not due to this branch.

**Pull request merged**